cluster/userdata-master.yaml

```diff
@@ -166,11 +166,11 @@ write_files:
 
       KUBECTL="docker run --net host -v /srv/kubernetes/manifests:/manifests:ro quay.io/coreos/hyperkube:v1.4.4_coreos.0 /hyperkube kubectl"
 
-      for manifest in kube-dns-de.yaml kube-dashboard-de.yaml heapster-de.yaml secretary-de.yaml mate-de.yaml; do
+      for manifest in kube-dns-de.yaml kube-dashboard-de.yaml heapster-de.yaml secretary-de.yaml mate-de.yaml prometheus-node-exporter-de.yaml; do
```

Comment:
Maybe create subfolders like `deployments` and `services`? The "-de" suffix also makes people wonder what it means (I thought about the language/country code :smirk: ))